### DISCUSSION TITLE
Run flexget via cli when the daeomn is running

### DISCUSSION THREAD

**Discussion by effemmeffe** [Flexget/Flexget#2845] [Q&A]
I have flexget running as daemon on my ubuntu box.
I use the scheduler plugin to run the tasks at certain time of the day.
But sometimes I also want to run some task by hand, via the cli.
SInce some of my tasks are very time consuming often it happens that when I want to run flexget by hand the daemon is already running some taks so I get a message saying that `There is a FlexGet process already running for this config, sending execution there` and the cli command does not execute until the task run by the daemon won't end.
If I want to immediately execute the task via the cli the only solution I found is to sto the daemon, tun the cli command and then restart the daemon.
Is there a more elegant solution for this?

**Comment by ksurl** [comment]
Not really. You could change to a cronjob so each run would be its own process but it may not have expected results with simultaneous writes to the sqlite db. There is also the config lock file. Maybe you need tighten up the config for long running tasks so they're completed faster

**Reply by effemmeffe** [reply]
The latter would be my preferred solution, but I don't really understand how flexget works, I copied my config from Jonybat and made some minor tweaks, but I'm not able to change it to be faster.
Thanks anyway.

**Comment by marp188** [comment]
Go to your .flexget config folder and remove the .config-lock file
